src/Protocol/Diaspora.php

```diff
@@ -2651,15 +2669,16 @@ private static function storePhotoAsMedia(int $uriid, $photo)
 	/**
 	 * Receives status messages
 	 *
-	 * @param array            $importer Array of the importer user
-	 * @param SimpleXMLElement $data     The message object
-	 * @param string           $xml      The original XML of the message
-	 * @param bool             $fetched  The message had been fetched and not pushed
+	 * @param array            $importer  Array of the importer user
+	 * @param SimpleXMLElement $data      The message object
+	 * @param string           $xml       The original XML of the message
+	 * @param int              $direction Indicates if the message had been fetched or pushed
```

Comment:
Please add a reference to the constants the parameter is taking its values from.